xbmc/Application.cpp

```diff
@@ -807,7 +807,7 @@ bool CApplication::CreateGUI()
 
   uint32_t sdlFlags = 0;
 
-#if (defined(HAS_SDL_OPENGL) || (HAS_GLES == 2)) && !defined(HAS_GLX)
+#if defined(TARGET_DARWIN_OSX)
```

Comment:
@Memphiz could you check this please. I don't think any or our GLES platforms other than iOS could have made use of this